src/main/java/io/rpg/config/model/GameObjectConfig.java

```diff
@@ -0,0 +1,37 @@
+package io.rpg.config.model;
+
+import io.rpg.model.data.Position;
+import io.rpg.model.object.GameObject;
+import io.rpg.model.object.GameObjects;
+import io.rpg.util.Result;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents {@link io.rpg.model.object.GameObject} configuration provided by user
+ * in configuration files.
+ */
+public class GameObjectConfig extends GameObject {
```

Comment:
Napewno GameObjectConfig powinien extendować GameObject?

Reply:
Semantycznie to nie jest najpiękniejsze rozwiązanie, ale wszystkie pola które były w `GameObject` muszą się też znaleźć w `GameObjectConfig` --> po prostu odziedziczyłem.